main.py

```diff
@@ -1,116 +1,99 @@
-# Import Pyton Libraries ---------------------------------------------------
-import os
+#!/usr/bin/env python3
+from os import path, getenv, listdir
 import subprocess
+import logging
+import argparse
 
-# Logging ------------------------------------------------------------------
```

Comment:
Logging? Logging what? I see no login going on.

Let's get rid of that amazing "documentation".